Makefile

```diff
@@ -0,0 +1,50 @@
+FEATURES_MINIMAL = --no-default-features --features 'std cargo' -p clap:3.0.0-beta.5
+FEATURES_FULL = --features 'wrap_help yaml regex unstable-replace unstable-multicall unstable-grouped'
+
+.PHONY: features-minimal features-full lint-minimal lint-full fmt-check lint coverage bench tests test debug clean
+
+features-minimal:
+	@echo "$(FEATURES_MINIMAL)"
+
+features-full:
+	@echo "$(FEATURES_FULL)"
```

Comment:
Should we prefix these feature names with `list-`?  A couple of times, I was thrown off by the names when looking around this PR.